cumulusci/robotframework/SalesforcePlaywright.py

```diff
@@ -173,14 +174,14 @@ def wait_until_salesforce_is_ready(
                 # No errors? We're golden.
                 break
 
-            except Exception:
+            except Exception as exc:
                 # dang. Maybe we landed somewhere unexpected?
                 if self._check_for_classic():
                     continue
 
                 if time.time() - start_time > timeout_seconds:
                     self.browser.take_screenshot()
-                    raise Exception("Timed out waiting for a lightning page")
+                    raise Exception("Timed out waiting for a lightning page") from exc
```

Comment:
How does it get into this block if there is no timeout?

Is the underlying exception CAUSING a timeout?

Or is it failing instantly and yet ending up in this block for some reason.